src/crimson/osd/ops_executer.cc

```diff
@@ -111,8 +111,8 @@ OpsExecuter::call_errorator::future<> OpsExecuter::do_op_call(OSDOp& osd_op)
   );
 }
 
-static watch_info_t create_watch_info(const auto& osd_op,
-                                      const auto& msg)
+static watch_info_t create_watch_info(const OSDOp& osd_op,
```

Comment:
During the review of watch / notify, I've forgotten the issue is not only about `auto` in return type, but also in parameters.
Thanks for reporting this, @ronen-fr!

Reply:
ahhh, thought you fixed it =)

Reply:
me too 8-)